contrib/localnet/orchestrator/smoketest/main.go

```diff
@@ -295,8 +303,9 @@ func LocalSmokeTest(_ *cobra.Command, _ []string) {
 	smokeTest.TestMessagePassingRevertSuccess()
 	smokeTest.CheckZRC20ReserveAndSupply()
 
-	smokeTest.TestPauseZRC20()
-	smokeTest.CheckZRC20ReserveAndSupply()
+	//DISABLED temporarily since older binaries might not work with this due to admin account
```

Comment:
I remember Bertrant created these two tests and he made them working on develop?

Reply:
I think `TestPauseZRC20` and `TestUpdateBytcode` can be commented out since it only concerns isolated actions and don't need to be checked everytime in the CI.

But I added in the genesis generation of smoke tests the account for foreign coin admin, I'm not sure what would cause it to not work then

Reply:
I think your change is not included in older versions. This particular test path would run the smoketest on an older version, do an upgrade then run the smoketest again. It isn't guaranteed that the account will be present in the older versions.

Reply:
We should comment `smokeTest.TestUpdateBytecode()` for the same reason, also use the admin account.

I think we could keep those disabled since it represents tests for administration, we might want to have a specific smoketest for it.